welcometothemoon.js

```diff
@@ -681,6 +681,14 @@ define([
         })
       },
 
+      onEnteringStateFactoryUpgrade(args) {
```

Comment:
should be useless with GenericPickSlot action

Reply:
Removed